config/feeds_config.json

```diff
@@ -66,7 +70,33 @@
       },
       "resources": {
         "yf_symbol": "EURUSD=X"
-      }
+      },
+      "value_type": "Numerical",
+      "aggregate_type": "Median"
+    },
+    {
+      "id": 222,
+      "name": "Text",
+      "fullName": "Some text",
+      "description": "Hello World",
+      "decimals": 8,
+      "report_interval_ms": 30000,
+      "quorum_percentage": 1,
+      "type": "Forex",
+      "script": "YahooFinance",
+      "pair": {
+        "base": "EUR",
+        "quote": "USD"
```

Comment:
Can we remove this?

Reply:
No, smart contract deployment expects some data here